recipes/nrlmsise-00/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "nrlmsise" %}
+{% set version = "00" %}
+{% set git_rev = "bc9a2feba4344e74201281e563332688a4d09cc3" %}
+{% set sha256 = "85044ca171431ef0044c5c4d62d5124d1af2dd632cff85203cecf682bfe5711a" %}
+{% set build_number = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://git.linta.de/~brodo/nrlmsise-00.git/snapshot/nrlmsise-00-{{ git_rev }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - 0001_new_path_include_for_conda_package_test.patch
+
+build:
+  number: {{ build_number }}
```

Comment:
```suggestion
{% set git_rev = "bc9a2feba4344e74201281e563332688a4d09cc3" %}

package:
  name: {{ name|lower }}
  version: {{ version }}

source:
  url: https://git.linta.de/~brodo/nrlmsise-00.git/snapshot/nrlmsise-00-{{ git_rev }}.tar.gz
  sha256: 85044ca171431ef0044c5c4d62d5124d1af2dd632cff85203cecf682bfe5711a
  patches:
    - 0001_new_path_include_for_conda_package_test.patch

build:
  number: 0
```

Reply:
You made suggested changes that were unrelated to your comment, but yes I understand the changes but do not see this still do not see this as blocking.